.github/workflows/statistics-daily.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: Statistics Daily
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 8 * * 2-5'  # Tuesdays to Fridays
```

Comment:
:thinking: Is the idea that we can't do Monday because Sunday there are no builds? I mean, we still have some builds, but :shrug: I think it might OK, but we can also wait.

Reply:
Yea ideally I would want Monday to look back to Friday (maybe incl Sat+Sun).